deps/wazuh_testing/wazuh_testing/qa_docs/lib/config.py

```diff
@@ -32,153 +36,194 @@ def __init__(self, *args):
+        else:
+            Config.LOGGER.warning('Cannot read test info fields')
     
-    def _read_module_info(self):
-        '''
-        brief: Reads from the config file the keys to be printed from module info
-        '''
-        if 'Module info' in self._config_data:
-            self.module_info = self._config_data['Module info']
+    def __read_module_info(self):
+        """Reads from the config file the fields to be printed from test info.
 
-    def _read_project_path(self):
-        """
-        brief: Reads from the config file the path of the project.
+        This functionality is used to print any custom field(s) you want.
+        You can use it if you only need a few fields to parse when a single test is run.
+        
+            For example you have this in your config.yaml:
+
+            Module info:
+                - test_system: os_platform
+                - test_vendor: os_vendor
+                - test_version: os_version
+                - test_target: component
         """
-        if 'Project path' in self._config_data:
-            self.project_path = self._config_data['Project path']
+        Config.LOGGER.debug('Reading module info from config file')
+
+        if 'Module info' in self._config_data:
+            self.module_info = self._config_data['Module info']
+        else:
+            Config.LOGGER.warning('Cannot read module info fields')
 
-    def _read_documentation_path(self):
+    def __set_documentation_path(self, path):
         """
-        brief: Reads from the config file the path of the documentation output.
+        brief: Sets the path of the documentation output.
         """
-        if 'Output path' in self._config_data:
-            self.documentation_path = self._config_data['Output path']
+        Config.LOGGER.debug('Setting the path documentation')
+        self.documentation_path = path
 
-    def _read_include_paths(self):
+    def __read_include_paths(self):
         """
         brief: Reads from the config file all the paths to be included in the parsing.
         """
+        Config.LOGGER.debug('Reading include paths from config file')
+
+        # Will be replaced by --type --module and --test , so you can run what you need
         if not 'Include paths' in self._config_data:
-            logging.error("Config include paths are empty")
-            raise Exception("Config include paths are empty")
+            raise QAValueError('Config include paths are empty', Config.LOGGER.error)
+
         include_paths = self._config_data['Include paths']
+
         for path in include_paths:
             self.include_paths.append(os.path.join(self.project_path, path))
 
-    def _read_include_regex(self):
+    def __read_include_regex(self):
         """
         brief: Reads from the config file the regexes used to identify test files.
         """
+        Config.LOGGER.debug('Reading the regular expressions to include files from config file')
```

Comment:
```suggestion
        Config.LOGGER.debug('Reading the regular expressions from the config file to include test files')
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523